### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
You probably just want to browse https://github.com/INDIGO-Initiative/database-app/tree/20210407docs/docs

File: docs/explanation/different-data-representations.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+* Single source of truth data
+* Single source of truth data  => Public and Sandbox API Data ( A one way conversion )
+* Single source of truth data  => Public Spreadsheet Data  ( A one way conversion )
+* Single source of truth data  <=> Private Spreadsheet Data  ( A two way conversion )
+
+## The differences between the API data and the data for the Spreadsheets
+
+There is no conceptual reason why the "Public API Data" and the "Spreadsheet Data" should be related at all.
+
+In practice, it turns out we want to do some of the same things to both sets of data.
```
Comment (Lathrisk) [inline docs/explanation/different-data-representations.md]: I think here we might not need `In practice` and `it turns out`. I'd probably go with `In practice we want to do some of the same things...`

File: docs/reference/vagrant_for_dev.md [hunk 2]
```diff
@@ -40,13 +42,15 @@ Add some `Types` records in the `Jsondataferret` section :
     
 ## Tests
 
-Run tests  (with Vagrant DB credentials):
+The Vagrant box setups up a second database, so running tests won't alter your current working data.
```
Comment (Lathrisk) [inline docs/reference/vagrant_for_dev.md]: `sets up` rather than `setups`?

**Comment by Lathrisk** [comment]
I did wonder if some of the sections could use bullet points or numbering for some of the steps but beside a couple minor comments it reads fine. I think it all makes sense, without having gone through the steps myself.

**Comment by ghost** [comment]
> I did wonder if some of the sections could use bullet points or numbering for some of the steps

The problem with this is then the steps each have to be one line, and roughly equal length. When you try and do this in situations where some of the steps need more detail (eg code samples) it can get iffy maybe? Maybe there is something else we can do to break up a long bit of text into steps so it's clearer for people, more headers maybe? But if ok, we'll leave that for later?

**Review by Lathrisk — APPROVED** [review]
Looks good, and yeah, just leave the formatting stuff till next time

**Pull request merged**